libelement/src/object_model/call_stack.cpp

```diff
@@ -11,6 +11,11 @@
 
 using namespace element;
 
+call_stack::call_stack()
+{
+    frames.reserve(100);
```

Comment:
Can you add a comment here to explain where 100 came from - so that people know whether they can change it in the future 🙈

Reply:
what does 🙈 mean?